src/lightning_lite/utilities/seed.py

```diff
@@ -0,0 +1,127 @@
+import logging
```

Comment:
The only function not getting moved is `isolate_rng`, which is why this isn't a clean file move. 
No implementation changes to the other functions.